common/pkgfiles/referencedfilelist.cpp

```diff
@@ -126,13 +126,38 @@ void splitDerivedDfsLocation(const char *address, StringBuffer &cluster, StringB
     prefix.append(basePrefix);
 }
 
+void splitDerivedDfsLocationOrRemote(const char *address, StringBuffer &cluster, StringBuffer &ip, StringBuffer &prefix,
```

Comment:
I don't suggest it for this PR (or even as a priority), but this function has too many parameters, with little logical ordering - if it was refactored it would be good to think about which parameters should be grouped together, and possibly moved into a class.